parabeac.dart

```diff
@@ -1,32 +1,54 @@
 import 'dart:convert';
 import 'dart:io';
+import 'package:args/args.dart';
 
+ArgResults argResults;
+
+// ignore: always_declare_return_types
 main(List<String> args) async {
-  List<String> arguments = ['lib/main.dart'];
-  final helpText = "Usage Options: \n" + "-url \n" + "-key \n" + "-Skey";
-  var url = '';
-  var key = '';
-  var sKey = '';
-  //If arguments is empty or only has -h
-  if (args.length == 0 || args[0] == '-h') {
-    print(helpText);
-  } else {
-    for (var i = 0; i < args.length; i += 2) {
-      switch (args[i]) {
-        case '-url':
-          url = args[i + 1];
-          break;
-        case '-key':
-          key = args[i + 1];
-          break;
-        case '-Skey':
-          sKey = args[i + 1];
-          break;
-        default:
-          arguments.addAll([args[i], args[i + 1]]);
-          break;
-      }
-    }
+  var arguments = <String>['package:parabeac_core/main.dart'];
```

Comment:
Thanks for making the previous adjustments! I think the last thing would be changing this from `package:parabeac_core/main.dart` to `lib/main.dart`. We tried testing it using this default value and it would always break unless we left the `lib/main.dart` 

However, I'd also like to know if this `package:parabeac_core/main.dart` worked for you🤔 . Thanks!

Reply:
Note: I'm using windows
It was working, however, after, I ran `flutter clean` right now and deleted the `./dart-tool` folder, it stopped working. `lib/main.dart` seems to consistently work